freqtrade/freqtradebot.py

```diff
@@ -919,8 +968,42 @@ def execute_sell(self, trade: Trade, limit: float, sell_reason: SellType) -> Non
         order = self.exchange.sell(pair=str(trade.pair),
                                    ordertype=ordertype,
                                    amount=trade.amount, rate=limit,
-                                   time_in_force=self.strategy.order_time_in_force['sell']
-                                   )
+                                   time_in_force=self.strategy.order_time_in_force['sell'])
+        order_id = order['id']
+        
+        fee = self.exchange._api.calculate_fee(symbol=trade.pair, type='', side='', amount=trade.amount, price=limit, takerOrMaker='maker')['cost']
+
+        if order.get('status', None) is None:
+            order['status'] = 'open'
+            order['cost'] = limit * trade.amount
+            order['price'] = limit
+            order['amount'] = trade.amount
+            order['side'] = 'sell'
+            order['type'] = 'limit'
+            order['is_open'] = True
+            order['filled'] = 0
+
+        if order.get('fee', None) is None:
+            order['fee'] = {'cost': fee, 'currency': trade.pair[:trade.pair.find('/')]}
```

Comment:
self.exchange.get_fee()

Reply:
exchange.get_fee returns only 'rate' for some reason

Reply:
ok, so we can use exchange._api.calculate_fee